### PR TITLE
Planning: updated open_space planner conf for park_and_go.

### PR DESCRIPTION
**Pull request by sjiang2018**

### CODE REVIEW

File: modules/planning/conf/scenario/park_and_go_config.pb.txt [hunk 1]
```diff
@@ -51,6 +51,21 @@ stage_config:{
...
+                        acc_weight: 1.0
+                        jerk_weight: 1.0
+                        kappa_penalty_weight: 100.0
+                        ref_s_weight: 100.0
+                        ref_v_weight: 0.0
+                        }
+                    }
+                }
+            }
+        }
```
Comment (xiaoxq) [inline modules/planning/conf/scenario/park_and_go_config.pb.txt]: Watch out indents.

File: modules/planning/scenarios/util/util.cc [hunk 2]
```diff
@@ -262,7 +264,8 @@ bool CheckADCReadyToCruise(Frame* frame,
        std::fabs(common::VehicleStateProvider::Instance()
                      ->vehicle_state()
                      .linear_velocity()) < kMinSpeed) &&
-      !is_near_front_obstacle && heading_align_w_reference_line) {
+      !is_near_front_obstacle && heading_align_w_reference_line &&
+      std::fabs(adc_position_sl.l()) < 0.5) {
     return true;
   }
   return false;
```
Comment (xiaoxq) [inline modules/planning/scenarios/util/util.cc]:
```
if (a) {
  return true;
}
return false;
```
equals to
`return a;`

Comment (sjiang2018) [reply modules/planning/scenarios/util/util.cc]: Fixed. Thank you.

File: modules/planning/conf/scenario/park_and_go_config.pb.txt (hunk 1 above)
Comment (sjiang2018) [reply modules/planning/conf/scenario/park_and_go_config.pb.txt]: Fixed. Thank you.

File: modules/planning/scenarios/util/util.cc [hunk 3]
```diff
@@ -257,15 +259,13 @@ bool CheckADCReadyToCruise(Frame* frame,
...
-       std::fabs(common::VehicleStateProvider::Instance()
-                     ->vehicle_state()
-                     .linear_velocity()) < kMinSpeed) &&
-      !is_near_front_obstacle && heading_align_w_reference_line) {
-    return true;
-  }
-  return false;
+  return ((common::VehicleStateProvider::Instance()->gear() ==
+               canbus::Chassis::GEAR_DRIVE ||
+           std::fabs(common::VehicleStateProvider::Instance()
```
Comment (xiaoxq) [inline modules/planning/scenarios/util/util.cc]: Save `common::VehicleStateProvider::Instance()` to a local var.

Comment (sjiang2018) [reply modules/planning/scenarios/util/util.cc]: fixed. Thx!

**Pull request merged**